pyprep/utils.py

```diff
@@ -21,6 +21,29 @@ def _intersect(list1, list2):
     return list(set(list1).intersection(set(list2)))
 
 
+def _mat_round(x):
```

Comment:
I was thinking that maybe it was better using the whole "matlab" prefix instead of the "mat" one. I understand is more verbose but one could confuse "mat"  as being associated with a matrix.

In any case, this is just a minor suggestion, is not that relevant.